.west/config

```diff
@@ -0,0 +1,7 @@
+[manifest]
```

Comment:
I'm not able to initialize your project with `west init`. 

![image](https://github.com/user-attachments/assets/be62a110-45c1-49c7-8b78-c83845dca7d8)

The reason of that is this file. `.config` is the file that is the **result/output** of the command `west init`. West will generate the one based on the content of the of the `west.yml` file taking into account `self` and `projects` properties.